### DISCUSSION TITLE
Does the decodeFloat1c function need to be multiplied by 255?

### DISCUSSION THREAD

**Discussion by smbera** [foxglove/community#697] [Q&A]
**Description**

- Version: 1.44.0-dev
- Platform: Chrome 112.0.5615.121 （64 bit）
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native):

https://github.com/foxglove/studio/blob/main/packages/den/image/decodings.ts#L178

I want to known, does this line of code need to multiply by 255? Because I can display the picture when I don't multiply it by 255, but it shows a white background when I multiply it by 255. please confirm please, thank you

**Comment by defunctzombie** [comment]
@smbera Thanks for engaging with your question however this is not a bug. Please do not file bug issues for things that are not bugs.

<img width="808" alt="image" src="https://user-images.githubusercontent.com/84792/233084285-e79b850d-0c5f-4045-b37e-91eeb29f15e2.png">

If you have a question - use the question link to ask it. I've converted this bug report into a discussion.

**Comment by jtbandes** [comment]
Can you please provide some sample data that reproduces the problem, and explain how you recorded the sample data?
